src/config.rs

```diff
@@ -0,0 +1,104 @@
+use std::{collections::HashMap, ffi::OsStr, fs};
+
+use common::extensions::*;
+use microtemplate::render;
+use serde::{Deserialize, Deserializer};
+use uniquote::Quote;
+
+use crate::paths::PATHS;
+
+#[derive(Clone, Default, Debug)]
+pub struct ConfigContext;
+
+impl microtemplate::Context for ConfigContext {
+    fn get_field(&self, field_name: &str) -> &str {
+        match field_name {
+            "game_dir" => PATHS.game_str.as_str(),
+            _ => "",
+        }
+    }
+}
```

Comment:
this is a pretty odd solution, I think the struct used as the context should just contain a field for `game_dir`

Reply:
I was trying to figure out the best place to process the `Task.values` HashMaps using microtemplates. Essentially I made it static. I've since learned a bit more rust syntax and a few more idioms that could solve the issue.

How would you go about it? Iterate through the values in an inner loop (or function) called from `get_configs` function and replace the values instead of doing it during deserialization? Convert ConfigContext to a static?

Reply:
you can just initialize the `ConfigContext` whenever it's needed and pass it as a parameter,
I'd avoid statics unless they are necessary, like the `CMD_STR` statics purpose is to avoid re-running the initialization code, I don't think we need any other statics